.rubocop.yml

```diff
@@ -322,13 +322,3 @@ Style/SpecialGlobalVars:
   Enabled: false
 Style/SelectByRegexp:
   Enabled: false
-
-# TODO these were temporarily disabled during the Ruby 2.7 -> 3.1 upgrade
-# in order to keep the upgrade diff small, they will be enabled/fixed in
-# a follow-on PR.
-Naming/BlockForwarding:
-  Enabled: false
```

Comment:
Will probably need to pull this change out to a separate PR from the freeze one... it generates a lot of changes.